src/routes/+layout.server.ts

```diff
@@ -1,6 +1,24 @@
 import { runtimeConfig, runtimeConfigUpdatedAt } from '$lib/server/runtime-config';
+import { CUSTOMER_ROLE_ID } from '$lib/types/User';
 
 export async function load(event) {
+	let viewportWidth;
+	if (runtimeConfig.viewportFor === 'everyone') {
+		viewportWidth = 'width=device-width';
+	} else if (
+		event.locals.user?.roleId !== CUSTOMER_ROLE_ID &&
+		runtimeConfig.viewportFor === 'employee'
+	) {
+		viewportWidth = 'width=device-width';
+	} else if (
+		event.locals.user?.roleId === CUSTOMER_ROLE_ID &&
+		runtimeConfig.viewportFor === 'visitors'
+	) {
+		viewportWidth = 'width=device-width';
+	} else {
+		viewportWidth = runtimeConfig.viewportContentWidth;
+	}
```

Comment:
use a `switch`:

```ts
const viewPortWidth = () => ({
  switch (runtimeConfig.viewportFor) {
    case ...: return ... ? ... : ...;
    ...
  }
})()
```

`switch` has better type-checking.

Also makes the logic clearer, I think some cases are wrong with the current logic (for example `'visitors` should include when `roleId` is `undefined`)